src/Private/Console.ps1

```diff
@@ -0,0 +1,987 @@
+            $flags += 'DualMode'
+        }
+
+        if ($disabled -and ('HTTP', 'HTTPS' -contains $protocol)) {
+            $flags += 'Disabled'
+        }
+        $flagString = if ($flags.Length -gt 0) { "[$($flags -join ',')]" } else { [string]::Empty }
+
+        # Display endpoint details
+        Write-PodeHost "   - $protocolLabel : $($_.Url) `t$flagString" -ForegroundColor $endpointsColor -Force:$Force
+    }
+
+    # Footer
+    Write-PodeHost
+
+    # Write a horizontal divider line to the console.
+    Write-PodeHostDivider -Force $true
+}
+
+<#
+.SYNOPSIS
+    Displays metrics for the Pode server in the console.
+
+.DESCRIPTION
+    This function outputs various server metrics, such as uptime and restart counts,
+    to the Pode console with styled colors based on the Pode context. The function
+    ensures a visually clear representation of the metrics for quick monitoring.
+
+.EXAMPLE
+    Show-PodeConsoleMetric
+
+    This command displays the Pode server metrics in the console with the
+    appropriate headers, labels, and values styled using Pode-defined colors.
+
+.NOTES
+    This function depends on the PodeContext and related server configurations
+    for retrieving metrics and console colors. Ensure that Pode is running and
+    configured correctly.
+
+.OUTPUTS
+    None. This function writes output directly to the console.
+
+#>
+function Show-PodeConsoleMetric {
+    # Determine the color for the metrics header
+    if ($null -ne $PodeContext.Server.Console.Colors.EndpointsHeader) {
+        $headerColor = $PodeContext.Server.Console.Colors.MetricsHeader
+    }
+    else {
+        $headerColor = [System.ConsoleColor]::White
+    }
+
+    # Determine the color for the metrics labels
+    if ($null -ne $PodeContext.Server.Console.Colors.Endpoints) {
+        $labelColor = $PodeContext.Server.Console.Colors.MetricsLabel
+    }
+    else {
+        $labelColor = [System.ConsoleColor]::Yellow
+    }
+
+    # Determine the color for the metrics values
+    if ($null -ne $PodeContext.Server.Console.Colors.Endpoints) {
+        $valueColor = $PodeContext.Server.Console.Colors.MetricsValue
+    }
+    else {
+        $valueColor = [System.ConsoleColor]::Green
+    }
+
+    # Write a horizontal divider line to separate the header
+    Write-PodeHostDivider -Force $true
+
+    # Write the metrics header with the current timestamp
+    Write-PodeHost "$($Podelocale.serverMetricsMessage) [$(Get-Date -Format 'yyyy-MM-dd HH:mm:ss')]" -ForegroundColor $headerColor
+
+    # Write another horizontal divider line for separation
+    Write-PodeHostDivider -Force $true
+
+    # Display the total uptime
+    Write-PodeHost $Podelocale.totalUptimeMessage -ForegroundColor $labelColor -NoNewLine
+    Write-PodeHost (Get-PodeServerUptime -Total -Readable -OutputType Verbose -ExcludeMilliseconds) -ForegroundColor $valueColor
+
+    # If uptime exceeds 1000 seconds, display uptime since last restart
+    if ((Get-PodeServerUptime) -gt 1000) {
```

Comment:
Why not just check the restart count here?
```powershell
if ((Get-PodeServerRestartCount) -gt 0) {
```

On these messages as well, they could do with a space after the `:`, as it looks a little cramped currently:

```
Total Uptime:6 minutes 34 seconds
Uptime Since Last Restart:4 minutes 7 seconds
Total Number of Restart:1
```

Reply:
because sometime I'm not thinking